docs/documentation/release-notes.md

```diff
@@ -60,16 +58,16 @@ For single line RNs, follow this format:
 ```markdown
 #### Integrations
 ##### Cortex XDR - IR
-  - Release note here.
+Release note here.
```

Comment:
@tschanfeld please revert this change to be a list (`- Release note here.`), as it matters for automated tests this document describes.

Reply:
@kgal-pan please check whether this passes validations. If it does not, let's change it back.